### PR TITLE
- extending the documentation for the ingress how-to

### PR DESCRIPTION
**Pull request by gambol99**

### CODE REVIEW

File: how-to-docs/ingress.md [hunk 1]
```diff
@@ -10,27 +11,34 @@ The annotation ```kubernetes.io/ingress.class: "nginx-internal"``` is used to sp
 
 In the following example the terms "myapp" and "myproject" have been used, these will need to be changed to the relevant names for your project. Where internal is used, this can be changed for an external ingress - everything else stays the same.
 
+```shell
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    # used to select which on the ingress this resource should be configured on
```
Comment (jits) [inline how-to-docs/ingress.md]: "which on the ingress" – could be phrased better?

File: how-to-docs/ingress.md [hunk 2]
```diff
@@ -10,27 +11,34 @@ The annotation ```kubernetes.io/ingress.class: "nginx-internal"``` is used to sp
...
+          servicePort: 8000
+        path: /
+  tls:
+  - hosts:
+    - "myapp.myproject.homeoffice.gov.uk"
+    # the name of the kubernetes secret in your namespace with tls.crt and tls.key
+    secretName: myapp-github-internal-tls
+```
 
+> Always ensure you are using TLS between the ingress controller and your pods by placing the annotation: "ingress.kubernetes.io/secure-backends": "true". At the moment; this is not enforce though there are plans to enforce by policy.
```
Comment (jits) [inline how-to-docs/ingress.md]: `enforce` -> `enforced`

File: how-to-docs/ingress.md [hunk 3]
```diff
@@ -10,27 +11,34 @@ The annotation ```kubernetes.io/ingress.class: "nginx-internal"``` is used to sp
...
-               servicePort: 8000
-             path: /
-   tls:
-     - hosts:
-         - "myapp.myproject.homeoffice.gov.uk"
-       secretName: myapp-github-internal-tls
-   ```
+#### **Using LetsEncrypt with Ingress**
+
+Assuming you are not bringing your own certificates, LetsEncrypt can be used to acquire certificates for both internal *(behind vpn NOT cluster TLS certs)* and external certificates. Simply place the annotation ```stable.k8s.psg.io/kcm.class: default``` into the ingress resource; A full list of the supported features can be found at [here](https://github.com/PalmStoneGames/kube-cert-manager/blob/master/docs/ingress.md). Note at present we ONLY allow you to request certificates via the ingress resource, not by the third part resource.
```
Comment (jits) [inline how-to-docs/ingress.md]: `found at [here]` -> ` found [here]`

`part` -> `party`

Comment (jaykeshur) [reply how-to-docs/ingress.md]: its a label, not a annotation:

```metadata:
  labels:
    stable.k8s.psg.io/kcm.class: default```

**Pull request merged**